ipp_macro_series_parser/agregats_transports/transports_parser.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jul 17 15:01:56 2015
+
+@author: thomas.douenne
+"""
+
+import os
+import pkg_resources
+import pandas as pd
+import numpy as np
+
+from ipp_macro_series_parser.config import Config
+
+parser = Config(
+    config_files_directory = os.path.join(pkg_resources.get_distribution('ipp-macro-series-parser').location)
+    )
+cn_directory = parser.get('data', 'cn_directory')
+
+
+def transports_parser(excelfile_name, onglet):
+    data_frame = pd.read_excel(excelfile_name, sheetname = onglet, skiprows = 2)
+    data_frame.rename(columns = {'Unnamed: 0': 'index'}, inplace = True)
+    data_frame = data_frame.dropna(thresh = 3)
+    data_frame.fillna('-', inplace = True)
+    return data_frame
+
+
+def transports_parser_categ(excelfile_name, onglet):
+    data_frame = pd.read_excel(excelfile_name, sheetname = onglet, skiprows = 2)
+    data_frame.rename(columns = {'Unnamed: 0': 'index'}, inplace = True)
+    data_frame['categorie'] = np.nan
+    data_frame.loc[data_frame[2005].isnull(), 'categorie'] = \
+        data_frame.loc[data_frame[2005].isnull(), 'index']
+    data_frame['categorie'].fillna(method='ffill', inplace = True)
+    data_frame = data_frame.dropna(thresh = 3)
+    return data_frame
+
+
+def transports_parser_souscateg(excelfile_name, onglet):
+    data_frame = pd.read_excel(excelfile_name, sheetname = onglet, skiprows = 2)
+    data_frame.rename(columns = {'Unnamed: 0': 'index'}, inplace = True)
+    data_frame = data_frame.dropna(thresh = 3)
+    return data_frame
+
+
+a_a3_a = transports_parser('C:/Users/thomas.douenne/Documents/GitHub/ipp-macro-series-parser/agregats_comptes_transports/a-transport-et-activite-economique.xls',
```

Comment:
Local file names should not appear. Use config_local.ini files to set them and parse it using ConfigParser